projects/igniteui-angular/src/lib/services/direction/directionality.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { TestBed } from '@angular/core/testing';
+import { Component } from '@angular/core';
+import { IgxDirectionality, DIR_DOCUMENT } from './directionality';
+
+interface FakeDoc {
+    body: { dir?: string };
+    documentElement: { dir?: string };
+}
+
+describe('IgxDirectionality', () => {
+    let fakeDoc: FakeDoc;
+    beforeEach(() => {
```

Comment:
Technically should be wrapped in `async` due to `compileComponents`
```suggestion
    beforeEach(async(() => {
```